.editorconfig

```diff
@@ -0,0 +1,38 @@
+# https://editorconfig.org/
```

Comment:
I think it's fine that this is added, but it does seem outside the scope of this PR, and will only help a subset of our code contributors: those using IDEs or those who have downloaded a plugin.

Reply:
@christinaholtNOAA sorry about that.  I was adding to my fork without realizing it would come up in the PR.